sphinx/ext/autodoc/__init__.py

```diff
@@ -255,6 +255,32 @@ def __getattr__(self, name: str) -> Any:
             return None
 
 
+class ObjectMember(tuple):
```

Comment:
I admit this is ugly. But it is needed to keep compatibility. But such container is needed for us to bring object information between Documenter's method (ex. `get_object_members()`, `filter_members()`, `sort_membes()` and so on).

In the next step, I'd like to deprecate old-styled tuples.